search/search_service/proxy/elasticsearch.py

```diff
@@ -61,6 +65,18 @@ class ElasticsearchProxy(BaseProxy):
         'tag': 'tags',
     }
 
+    # mapping to translate request for feature resources
+    FEATURE_MAPPING = {
```

Comment:
does key need to be here now as well?

Reply:
or guess not cause it's not used to match results

Reply:
right